### PR TITLE
#167592012 Add descriptive readme

### PR DESCRIPTION
**Pull request by devPinheiro**
#### What does this PR do?
- It adds descriptive readme 

#### Description of Task to be completed?
- update readme file

#### Any background context you want to provide?
none

#### What are the relevant pivotal tracker stories?
- PT Story link - `[#167592012 ] (https://www.pivotaltracker.com/story/show/167592012 )`

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
-A Social platform for the creative at heart.
+# Getting started
+
+This project contains frontend implementation for the Author's Haven project. If you are interested in checking out the backend, please do follow this link:
+[persephone-ah-backend](https://github.com/andela/persephone-ah-backend)
+
+Start with the following instructions
+
+Clone repo to your local machine and chnage directory into the folder
+
```
Comment (tunedev) [inline readme.md:11]: Hi @devPinheiro, good job with the readme file, I noticed a typo error, 'chnage' I think it should be 'change'

Comment (devPinheiro) [reply readme.md:11]: I have fixed this. Thanks for pointing it out.

**Review by fxola — APPROVED** [review]
Looks good to me

**Review by tunedev — APPROVED** [review]
LGTM.

**Pull request merged**